server/sockets/index.js

```diff
@@ -70,16 +71,18 @@ const socketListenTo = server => {
       cookieParser
     })
   );
+};
+
+const getSocketInstance = () => {
+  if (!socketInstance) {
+    throw new Error('Socket not initialized...');
+  }
 
-  const cohortViewClients = new Map();
-  const allCohortsViewClients = new Map();
-  const dashboardViewClients = new Map();
-  const listViewClients = new Map();
-  const cohortClientLocks = new Map();
-  const itemClientLocks = new Map();
-  const listClientLocks = new Map();
+  return socketInstance;
```

Comment:
could we return socket instead of socketInstance to not use io.socket everywhere but only socket?

Reply:
I've tried to export just `return socketInstance.sockets ` but it doesn't work.